frontend/tests/unit/example.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from "@vue/test-utils";
```

Comment:
This is an example right? best to remove it.

Reply:
I have removed the file